src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -50,6 +51,7 @@ public Command filterByCommandType(String[] commandComponents) throws Exception
             command = new ExitCommand();
             break;
         default:
+            AddInstrumentParser.logger.info(LogHelper.LOG_INVALID_COMMAND);

```

Comment:
Given this, maybe shift the logger attribute from addinstrument parser to inputparser instead